examples/js/loaders/ColladaLoader.js

```diff
@@ -3886,11 +3903,11 @@ THREE.ColladaLoader.prototype = {
 		var textureLoader = new THREE.TextureLoader( this.manager );
 		textureLoader.setPath( this.resourcePath || path ).setCrossOrigin( this.crossOrigin );
 
-		var tgaLoader;
+		var tgaLoader = this.tgaLoader;
 
-		if ( THREE.TGALoader ) {
+		if ( tgaLoader ) {
 
-			tgaLoader = new THREE.TGALoader( this.manager );
+			if ( tgaLoader.manager === THREE.DefaultLoadingManager ) tgaLoader.manager = this.manager;
```

Comment:
Even this feels a bit unpredictable to me, but less so than before. Consider:

```js
var tgaLoader = new THREE.TGALoader();
var modelLoader1 = new THREE.ColladaLoader( manager1 );
modelLoader1.setTGALoader( tgaLoader );

var modelLoader2 = new THREE.ColladaLoader( manager2 );
modelLoader2.setTGALoader( tgaLoader );

modelLoader2.load( ... );
setTimeout(() => modelLoader1.load( ... ), 500);
```

... which manager will TGALoader use? I suspect it will use both, and not in a particularly reliable way, depending on network timing. 🙃 If the manager must be overridden at all, I think it would be best to do that in `setTGALoader` or `setLoadingManager`.

Reply:
Um, we could also just inject just the class definition instead of an object. Something like:
```
loader.setTGALoader( THREE.TGALoader );
```
In this way, we avoid all issues discussed before.

Reply:
In this case yes, but in general no. Other dependencies like DRACOLoader or BasisTextureLoader would require configuration, and we can't do that without passing an instance. 

I don't really see a reason that the model loader's manager needs to be magically applied to the texture loader... Would you also override setPath and setResourcePath for loaders that have them? Could we just consider TGALoader pre-configured for now?

Reply:
Okay, I'm fine with that 👍